README.md

```diff
@@ -20,45 +24,44 @@ To succesfully deploy MPS, the following software must be installed on your deve
 
 To deploy the MPS on a local development system: 
 
-1. Clone the MPS repository on your development system.
+1. Clone the repo and switch to the `mps` directory.
     ```
     git clone https://github.com/open-amt-cloud-toolkit/mps.git && cd mps
     ```
 
-2. Modify the mps/.mpsrc file common name settings. Save and close the file.
-    ```
-    “common_name” : “the development system’s IP address”
-    ```
+2. Open the `.mpsrc` file to edit.
 
+3. Update the following 3 fields. Save and keep track of the values you choose.
 
-3. Navigate to mps/webui/src and modify the app.config.js file. Replace rpsServerIP and serverIP with your development system’s IP address. Save and close the file.
-    ```
-    const rpsServerIP = process.env.REACT_APP_RPS_SERVER ? process.env.REACT_APP_RPS_SERVER : '192.168.0.8'; 
-    const serverIP = process.env.REACT_APP_MPS_SERVER ? process.env.REACT_APP_MPS_SERVER : '192.168.0.8';
-    ```
+    | Field Name | Required | Usage |
+    | ------------- | ------------------ | ------------ |
+    | web_admin_user | Username of your choice | For logging into the Sample Web UI |
+    | web_admin_password | **Strong** password of your choice | For logging into the Sample Web UI |
+    | jwt_secret | A strong secret of your choice | Used when generating a JSON Web Token for authentication |
```

Comment:
Also mention if using Kong, Kong configuration needs to use use same jwt_secret in order to validate the token.